apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -110,13 +110,12 @@ public static IApplicationBuilder UseGoogleTrace(this IApplicationBuilder app)
             var serviceOptions = new TraceServiceOptions();
             setupAction(serviceOptions);
 
-            services.AddGoogleTrace(SetupCommonOptions);
-            void SetupCommonOptions(Common.TraceServiceOptions commonOptions)
+            services.AddGoogleTrace((Common.TraceServiceOptions commonOptions) =>
```

Comment:
Does the parameter need to be explicitly typed? I'd expect `services.AddGoogleTrace(options => { ... })` to work, but I may be missing something.

Reply:
Yes, because we now have `Common.CloudTraceExtensions.UseGoogleTrace(Action<Common.TraceServiceOptions>)` and `AspNetCore.CloudTraceExtensions.UseGoogleTrace(Action<AspNetCore.TraceServiceOptions>)`.
I'm calling the one in Common from the one in AspNetCore, if I don't type the parameter this will be a recursive call.